### ISSUE TITLE
Can't load latest version: can't find libgfortran.so.4

### ISSUE THREAD

**Issue by garrettjstevens** [conda-forge/r-bindrcpp-feedstock#4]
Hello,

Issue:
I installed this package in a brand-new env, but I can't load it in R.
Create:
```
conda create -y -n Rbindr r-bindrcpp
```

Run:
```
$ R
...
> library(bindrcpp)
Error: package or namespace load failed for ‘bindrcpp’ in dyn.load(file, DLLpath = DLLpath, ...):
 unable to load shared object '..../anaconda3/envs/Rbindr/lib/R/library/bindrcpp/libs/bindrcpp.so':
  libgfortran.so.4: cannot open shared object file: No such file or directory
```
<br/>
Environment (<code>conda list</code>):
<details>

```
# packages in environment at ..../anaconda3/envs/Rbindr:
#
# Name                    Version                   Build  Channel
bzip2                     1.0.6                h470a237_2    conda-forge
ca-certificates           2018.4.16                     0    conda-forge
cairo                     1.14.12              he56eebe_3    conda-forge
curl                      7.60.0               h84994c4_0
fontconfig                2.13.0               h074f895_3    conda-forge
freetype                  2.8.1                         0    conda-forge
gettext                   0.19.8.1                      0    conda-forge
glib                      2.55.0                        0    conda-forge
graphite2                 1.3.11               hfc679d8_0    conda-forge
gsl                       2.4                  h14c3975_4
harfbuzz                  1.8.4                h2bb21d5_0    conda-forge
icu                       58.2                 hfc679d8_0    conda-forge
jpeg                      9c                   h470a237_0    conda-forge
krb5                      1.16.1               hbb41f41_0    conda-forge
libcurl                   7.60.0               h1ad7b7a_0
libedit                   3.1.20170329         haf1bffa_0    conda-forge
libffi                    3.2.1                         3    conda-forge
libgcc-ng                 7.2.0                hdf63c60_3
libgfortran               3.0.0                         1
libiconv                  1.15                 h470a237_1    conda-forge
libpng                    1.6.34               ha92aebf_1    conda-forge
libssh2                   1.8.0                h5b517e9_2    conda-forge
libstdcxx-ng              7.2.0                hdf63c60_3
libtiff                   4.0.9                he6b73bb_1    conda-forge
libuuid                   1.0.3                         1    conda-forge
libxcb                    1.13                 h470a237_2    conda-forge
libxml2                   2.9.8                h422b904_2    conda-forge
ncurses                   6.1                           0    conda-forge
openssl                   1.0.2o                        0    conda-forge
pango                     1.41.0               hd475d92_0
pcre                      8.42                 h439df22_0
pixman                    0.34.0                        2    conda-forge
pthread-stubs             0.4                  h470a237_1    conda-forge
r-base                    3.4.1                hff14c37_6    conda-forge
r-bindr                   0.1.1                  r3.4.1_0    conda-forge
r-bindrcpp                0.2.2            r341h9d2a408_0    conda-forge
r-plogr                   0.2.0            r341h6115d3f_0    conda-forge
r-rcpp                    0.12.17          r341h9d2a408_0    conda-forge
readline                  7.0                  haf1bffa_1    conda-forge
tk                        8.6.8                         0    conda-forge
xorg-kbproto              1.0.7                h470a237_2    conda-forge
xorg-libice               1.0.9                h470a237_4    conda-forge
xorg-libsm                1.2.2                h470a237_4    conda-forge
xorg-libx11               1.6.5                h470a237_2    conda-forge
xorg-libxau               1.0.8                h470a237_6    conda-forge
xorg-libxdmcp             1.1.2                h470a237_7    conda-forge
xorg-libxext              1.3.3                h470a237_4    conda-forge
xorg-libxrender           0.9.10               h470a237_2    conda-forge
xorg-renderproto          0.11.1               h470a237_2    conda-forge
xorg-xextproto            7.3.0                h470a237_2    conda-forge
xorg-xproto               7.0.31               h470a237_7    conda-forge
xz                        5.2.4                h14c3975_4
zlib                      1.2.11               h470a237_3    conda-forge
```
</details>

<br/>
Details about  <code>conda</code> and system (<code>conda info</code>):
<details>

```
     active environment : Rbindr
    active env location : ..../Rbindr
            shell level : 1
       user config file : ..../.condarc
 populated config files : ..../.condarc
          conda version : 4.5.4
    conda-build version : 3.10.5
         python version : 3.6.5.final.0
       base environment : ..../anaconda3  (writable)
           channel URLs : https://conda.anaconda.org/bioconda/linux-64
                          https://conda.anaconda.org/bioconda/noarch
                          https://conda.anaconda.org/conda-forge/linux-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.continuum.io/pkgs/main/linux-64
                          https://repo.continuum.io/pkgs/main/noarch
                          https://repo.continuum.io/pkgs/free/linux-64
                          https://repo.continuum.io/pkgs/free/noarch
                          https://repo.continuum.io/pkgs/pro/linux-64
                          https://repo.continuum.io/pkgs/pro/noarch
          package cache : ..../anaconda3/pkgs
                          ..../.conda/pkgs
       envs directories : ..../anaconda3/envs
                          ..../.conda/envs
               platform : linux-64
             user-agent : conda/4.5.4 requests/2.18.4 CPython/3.6.5 Linux/4.9.27-14.31.amzn1.x86_64 amzn/2017.03 glib
                UID:GID : 0:0
             netrc file : None
           offline mode : False
```
</details>

Note: some paths masked with `....` for security

I do seem to have `libgfortran` in the environment from some dependancy, but I'm not sure where it's from, and the latest version in conda-forge seems to be v3, and I think this is looking for v4?

Please let me know if I can provide anything else. Thanks!

**Comment by jdblischak** [comment]
@garrett-stevens Thanks for reporting this. It is presumably due to the recently merged changes from PR #3.

I tried recreating the issue using the conda-forge Docker image linux-anvil, but it worked in this case:

```
sudo docker run --rm -it condaforge/linux-anvil
conda create -y -n Rbindr r-bindrcpp
source activate Rbindr
Rscript -e "library(bindrcpp)"
```

Could you please update to the latest version of conda (`conda update conda`) and try again? Also, commit https://github.com/conda-forge/r-bindrcpp-feedstock/commit/8ffd542b6df62f2f1dbc7e3202cba8566a1e1a8e may also have changed things depending on when exactly you last ran the code.

**Comment by garrettjstevens** [comment]
It looks like commit 8ffd542 might have fixed it, since now it works without even updating my conda version. Thanks!
